concepts/modules/.meta/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "Crystal has modules which can be used for namespacing and organizing code. It can also be used to create mixins and to extend classes.",
+  "blurb": "Crystal has modules which can be used for namespaces and organizing code. It can also be used to create mixins and to extend classes.",
```

Comment:
The existing language makes sense to me. What's your reasoning for making this change?

Reply:
I didn't really know it was a word or a version of namespaces. But I am reverting that change

Reply:
Got it, thanks!